build/media-suite_compile.sh

```diff
@@ -1570,9 +1570,9 @@ if  { ! mpv_disabled vapoursynth || enabled vapoursynth; }; then
             [vsscript-description]='Library for interfacing VapourSynth with Python'
             [vsscript-private]="-l$_python_lib -lstdc++"
         )
-        for _file in vapoursynth vsscript; do
-            gendef - "../$_file.dll" >/dev/null 2>&1 |
-                sed -r -e 's|^_||' -e 's|@[1-9]+$||' > "${_file}.def"
+        for _file in VapourSynth VSScript; do
```

Comment:
Be careful with these. Most unix tools expect case sensitivity, Windows doesn't mind it.

Reply:
Checked the original archive, the those are the actual cases.

Reply:
I was confused when I tried to tab autocomplete the name, but nothing came up

Reply:
I mean in the sense that FFmpeg's configure and mpv's configure will use the all lowercase names to look for the libnames. `gendef` and `dlltool` don't mind being given wrong-case files, since they're Windows tools.
Either that or, if you want to make code that's "linux-equivalent", use the all-lowercase to generate the new files, since that's the canonical name (same reason why vsscript-name is vapoursynth-script).

Reply:
I see, I see, going to just revert this part since adding another variable just for lowercased file name is probably not worth it at this point.

Maybe in the future if I decide to endeavor to make the suite work on linux. That's probably a long time away

Reply:
Nevermind, I didn't see it was fixed in a later commit.